epi/src/lib.rs

```diff
@@ -252,6 +252,11 @@ impl<'a> Frame<'a> {
         self.0.output.decorated = Some(decorated);
     }
 
+    /// Drag window
```

Comment:
This needs some documentation, maybe something like

```suggestion
    /// When called, the native window will follow the
    /// movement of the cursor while the primary mouse button is down.
    ///
    /// Does not work on the web, and works badly on Mac.
```

…because it does work very badly on my mac :)